site/_docs/java-tools.md

```diff
@@ -11,43 +11,136 @@ supports both the local file system and HDFS.
 
 The subcommands for the tools are:
 
-  * meta - print the metadata of an ORC file
-  * data - print the data of an ORC file
-  * scan (since ORC 1.3) - scan the data for benchmarking
   * convert (since ORC 1.4) - convert JSON files to ORC
+  * count (since ORC 1.6) - recursively find *.orc and print the number of rows
+  * data - print the data of an ORC file
   * json-schema (since ORC 1.4) - determine the schema of JSON documents
+  * key (since ORC 1.5) - print information about the encryption keys
+  * meta - print the metadata of an ORC file
+  * scan (since ORC 1.3) - scan the data for benchmarking
+  * version (since ORC 1.6) - print the version of this ORC tool
 
 The command line looks like:
 
 ~~~ shell
 % java -jar orc-tools-X.Y.Z-uber.jar <sub-command> <args>
 ~~~
 
+## Java Convert
+
+The convert command reads several JSON files and converts them into a
+single ORC file.
+
+-e,--escape `<escape>`
+  : Sets CSV escape character
```

Comment:
In HTML, `--escape` becomes `-escape`. Shall we use a different way to express this?
![Screen Shot 2021-08-31 at 8 18 59 AM](https://user-images.githubusercontent.com/9700541/131529938-d7b0adaf-7de1-46da-90b7-221048d41682.png)

Reply:
Use ``might work, I'll test it later

Reply:
Thanks!

Reply:
I've updated java-tools.md. My local preview html confirms that it's fixed.